sdcm/tester.py

```diff
@@ -836,7 +843,27 @@ def get_stress_results_bench(self, queue):
 
     def get_gemini_results(self, queue):
         results = self.loaders.get_gemini_results(queue)
-        return results
+        result = self.verify_gemini_results(results)
+        return result
+
+    def verify_gemini_results(self, results):
+        stats = {'status': None, 'results': []}
+        if not results:
+            self.fail('Gemini results are not found.')
```

Comment:
AFAIK `self.fail` will throw an exception so all the code below will be unreachable.

Reply:
replace with log.error